packages/elasticsearch/spec

```diff
@@ -2,7 +2,9 @@
 name: elasticsearch
 
 dependencies:
-  - openjdk-11
+
+- openjdk-11
 
 files:
-  - elasticsearch/elasticsearch-7.9.3-no-jdk-linux-x86_64.tar.gz
+
+- elasticsearch/elasticsearch-7.17.5-no-jdk-linux-x86_64.tar.gz
```

Comment:
Does the leading whitespace matter here?

Reply:
I don't know but I can remove it. My editor auto-added it